_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -43792,7 +43792,7 @@
 "nWB" = (
 /obj/structure/closet,
 /obj/item/clothing/under/suit/black,
-/obj/item/clothing/under/rank/civilian/lawyer/black/skirt,
+/obj/item/clothing/under/suit/black/skirt,
```

Comment:
Is this taking into account the washing machine interaction with lawyer suits

Reply:
Yep! Since the original 'plain black pants white buttondown' is now that service subtype of a GAGS item and wouldn't work as well, I opted to just change it to use the Black Suit (the ones in the top-right of New)
![image](https://github.com/tgstation/tgstation/assets/76465278/a3d0edea-a77d-4111-b661-4620dac8c8ef)

The Black Suit is now using the /obj/item/clothing/under/suit/black path instead since it was free, which means the washing machine change isn't listed (This is also why I didn't change the under/suit/black in the file this comment is on either; now it's just the full black suit by default. The skirt just needed to be repathed to follow it)

Reply:
I've re-used a path or two after removing other items, so now suit/black is actually used by the Two-piece suits
![image](https://github.com/tgstation/tgstation/assets/76465278/267a4037-96ef-451b-a945-adb92569c06c)
The washing machine code is unchanged and should now give these instead of the old item; since the old item is GAGS and has no skirt variant to match I figured it'd be better to just use these.
(This is why only the skirt has a change in the linked file too; it's just making it so both two-pieces are in the locker instead of very different suits)
(I still need to redo lawyer suits after this PR)